### ISSUE TITLE
sometimes garbled characters in language status item

### ISSUE THREAD

**Issue by Eskibear** [microsoft/vscode#159530]
The code to define the language status item is [HERE](https://github.com/redhat-developer/vscode-java/blob/29cd5729403e454c228a183f1f4d1b50d137df59/src/languageStatusItemFactory.ts#L55-L56).

Usually it looks well as  

![image](https://user-images.githubusercontent.com/2351748/187367322-243f6062-377e-4991-b4e0-b5db7d5379aa.png)

But today just after a reloading, the hyphen is garbled as below:

![image](https://user-images.githubusercontent.com/2351748/187367355-a0a96ed4-9a44-4f7c-923b-257910c4372f.png)

Not sure how to stably reproduce it just happened...Just FYI in case there might be some edge cases...

**Comment by jrieken** [comment]
We have seen this across the board with electron, esp after toggling dev tools. @Eskibear Did you do that by any chance? Had dev tools open and/or inspected some html elements?

**Comment by Eskibear** [comment]
> esp after toggling dev tools

Yes. I was doing smoke test against Java extension and I did toggle dev tools because I wanted to see if there was any console error. And for the first time everything was ok. After reloading for several times (not sure whether dev tools window was left open or not), I saw the garbled hyphen above.

**Comment by jrieken** [comment]
closing as upstream
